### ISSUE TITLE
Expose cleanslate as an npm package

### ISSUE THREAD

**Issue by seriousben** [premasagar/cleanslate#38]
This would make it usable within a react app super easily.

**Comment by ddelrio1986** [comment]
https://www.npmjs.com/package/cleanslate

I do use it within React app already.

**Comment by premasagar** [comment]
Thanks. I didn't know it was available on NPM. That's useful.
It looks like it's pointing to a fork of the repo. I'll ask if we can have it moved to point at the main repo.

**Comment by seriousben** [comment]
Yes, could we transition ownership of that npm package to this repo?
